list.go

```diff
@@ -0,0 +1,90 @@
+// +build linux
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+var listCommand = cli.Command{
+	Name:  "list",
+	Usage: "lists containers started by runc with the given root",
+	Action: func(context *cli.Context) {
+
+		// preload the container factory
+		if factory == nil {
+			err := factoryPreload(context)
+			if err != nil {
+				logrus.Fatal(err)
+				return
+			}
+		}
+
+		// get the list of containers
+		root := context.GlobalString("root")
+		absRoot, err := filepath.Abs(root)
+		if err != nil {
+			logrus.Fatal(err)
+			return
+		}
+		list, err := ioutil.ReadDir(absRoot)
+
+		w := tabwriter.NewWriter(os.Stdout, 12, 1, 3, ' ', 0)
+		fmt.Fprint(w, "ID\tPID\tSTATUS\tSTARTED\tBUNDLE\tROOTFS\n")
```

Comment:
Short answer... no.  

This line of code (41) merely caches up the column headings for the tab writer.  

With this list command I'm just loading up the containers and state and using the existing error checking that those apis provide.  The loaded containers may have just been created or started or paused, etc.   If there is no init pid stored away for the container the string should be empty and thus the column will be empty.  If we don't like that I could substitute no init pid in the state for the container ""... for "n/a"  or "none" or some other default text.  Let's see what it looks like when dug's create/start split drops.